<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>621</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="typeComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>200</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string comment="init" extracomment="init">初始化</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="in">车辆入场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="out">离场前</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="leave">离场后</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="orderQuery">订单查询</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>附件上传</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>订单补缴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>免费车辆</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="reqLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>请求参数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="replabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>返回值:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="typeLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>数据接口:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="requestPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>{
&quot;manufacturer&quot;:&quot;201808290001&quot;,
&quot;parkCode&quot;:&quot;201808150001&quot;,
&quot;sysCode&quot;:&quot;0001&quot;,
&quot;channelNumber&quot;:&quot;A4&quot;,
&quot;key&quot;:&quot;201822134513165313&quot;,
&quot;ssxUrl&quot;:&quot;http://192.168.1.11:8080/miniprogram-web&quot;
}</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="responsePlainTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>typeComboBox</tabstop>
  <tabstop>requestPlainTextEdit</tabstop>
  <tabstop>responsePlainTextEdit</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>submitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
